includes/core-changes/corefx/5.0/obsolete-consoleloggeroptions-properties.md

```diff
@@ -0,0 +1,129 @@
+### Obsolete properties on ConsoleLoggerOptions
+
+The <xref:Microsoft.Extensions.Logging.Console.ConsoleLoggerFormat?displayProperty=nameWithType> type and some properties on <xref:Microsoft.Extensions.Logging.Console.ConsoleLoggerOptions> are now obsolete.
+
+#### Change description
+
+Starting in .NET 5.0, the <xref:Microsoft.Extensions.Logging.Console.ConsoleLoggerFormat?displayProperty=nameWithType> type and several properties on <xref:Microsoft.Extensions.Logging.Console.ConsoleLoggerOptions> are obsolete. The obsolete properties are:
+
+- <xref:Microsoft.Extensions.Logging.Console.ConsoleLoggerOptions.DisableColors?displayProperty=nameWithType>
+- <xref:Microsoft.Extensions.Logging.Console.ConsoleLoggerOptions.IncludeScopes?displayProperty=nameWithType>
+- <xref:Microsoft.Extensions.Logging.Console.ConsoleLoggerOptions.TimestampFormat?displayProperty=nameWithType>
+- `UseUtcTimestamp`
+- <xref:Microsoft.Extensions.Logging.Console.ConsoleLoggerOptions.Format?displayProperty=nameWithType>
+
+With the introduction of new formatters, these properties are now available on the individual formatters.
+
+#### Reason for change
+
+The <xref:Microsoft.Extensions.Logging.Console.ConsoleLoggerOptions.Format> property is an enumeration type, which cannot represent a custom formatter.
+
+The remaining properties were set on <xref:Microsoft.Extensions.Logging.Console.ConsoleLoggerOptions> and applied to both of the built-in formats for console logs. However, with the introduction of a new formatter API, it makes more sense for formatting to be represented on the formatter-specific options. This change provides better separation between the logger and logger formatters.
+
+#### Version introduced
+
+5.0 Preview 8
+
+#### Recommended action
+
+- Use the new <xref:Microsoft.Extensions.Logging.Console.ConsoleLoggerOptions.FormatterName?displayProperty=nameWithType> property in place of the <xref:Microsoft.Extensions.Logging.Console.ConsoleLoggerOptions.Format?displayProperty=nameWithType> property. For example:
+
+  ```csharp
+  loggingBuilder.AddConsole(options =>
+  {
+    options.FormatterName = ConsoleFormatterNames.Systemd;
+  });
+  ```
+
+  There are several differences between <xref:Microsoft.Extensions.Logging.Console.ConsoleLoggerOptions.FormatterName> and <xref:Microsoft.Extensions.Logging.Console.ConsoleLoggerOptions.Format>:
```

Comment:
from looking at the preview link some of the xrefs in this section are looking a little odd in the text. If that is expected then ignore my comment here:

![image](https://user-images.githubusercontent.com/5897654/88855852-5b053400-d1a8-11ea-89c2-a91775ce71ae.png)

Reply:
Yes, it's expected because the preview 8 APIs aren't published yet.